.gitlab/functional_test/static_quality_gate.yml

```diff
@@ -51,6 +51,8 @@ static_quality_gates:
     - windows_msi_and_bosh_zip_x64-a7
  #  - job: agent_dmg-x64-a7
  #    optional: True
+ #  - job: agent_dmg-arm64-a7
+ #    optional: True
```

Comment:
Not sure whether we should add this here or rather remove the existing commented out bit altogether.